spec/models/page_spec.rb

```diff
@@ -1,11 +1,10 @@
 require 'spec_helper'
 
-describe Page do
-  after(:each) { Page.destroy_all }
+RSpec.describe Page do
   it 'must have a unique name' do
     expect(Page.new).to be_invalid
-    expect(Page.new({name: "Sports"})).to be_valid
-    Page.create!({name: "Sports"})
+    expect(Page.new({name: 'Sports'})).to be_valid
```

Comment:
Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.